moveit_ros/perception/mesh_filter/CMakeLists.txt

```diff
@@ -15,6 +15,8 @@ ament_target_dependencies(${MOVEIT_LIB_NAME}
   Eigen3
   Boost
 )
+target_link_libraries(${MOVEIT_LIB_NAME} ${gl_LIBS} GLUT::GLUT ${GLEW_LIBRARIES})
```

Comment:
Instead of ${GLEW_LIBRARIES}, I have found it's more reliable portability wise to use GLEW::glew